### ISSUE TITLE
Melda MComb crashing Carla

### ISSUE THREAD

**Issue by ZestieBumwhig** [robbert-vdh/yabridge#126]
This is not a "help request," just passing it along in case it's useful.

I am able to get a core dump with Melda MComb VST and VST3 in Carla. It only happens after a lot of knob twiddling. When I use Carla's native bridge, it doesn't seem to crash. Also, Carla didn't crash with other VST3s and even other Melda VST3s (thru yabridge)... so maybe this is so specific of a scenario to not be of any use to you! (this is carla-git from the AUR).

I didn't test the VST3 on Reaper, because I can't load Melda VST3s there for some reason... but that's for another thread.

$ carla
Carla 2.3.1 started, status:
  Python version: 3.9.6
  Qt version:     5.15.2
  PyQt version:   5.15.4
  Binary dir:     /usr/lib/carla
  Resources dir:  /usr/share/carla/resources
Frontend pixel ratio is 1.0
libjack.so.0 loaded successfully!
Aborted (core dumped)

**Comment by robbert-vdh** [comment]
Which Carla 2.3.1 commit? The regular Carla 2.3.1 release doesn't work with VST3 plugins because it will throw an X11 error when you try to open the editor. (fixed in https://github.com/falkTX/Carla/commit/b0f1b0530929cd94c9a583f6586eeef8f3d8ace4)

You will need to downgrade to Carla 2.3.0, wait for the next version to be released, or you can update to the very latest carla-git (VST3 plugins works fine there again, including MComb.vst3).

**Comment by ZestieBumwhig** [comment]
This is the latest commit, from the Arch AUR, of carla-git (I installed this based on our Discord discussion). It took a minute or so of GUI knob adjusting to happen, but then I started testing and it happened reliably (only with MComb).

HOWEVER this morning, doing the same tests, I can't get it to crash. So - I don't know what it was. Thought I found a useful bug, but it was just a hallucination - I'm seeing imaginary bugs like a Philip K Dick protagonist. Sorry.
